gitlab/README.md

```diff
@@ -0,0 +1,44 @@
+# Nimbella Commander GitLab Command Set
+A Nimbella Commander command set for viewing GitLab project statistics, creating tickets, and assigning issues.
+
+## Features
+- View community contribution stats for GitLab projects
+- Create a ticket for a GitLab project
+- Assign issues for a GitLab project
+- View list of users in project repository
+
+## Install
+```
+/nc csm_install gitlab
+```
+
+## Requirements
+```
+In order to use this command set, you must set up a Personal Access Token on GitLab with API access. And save the token as a secret with the key 'gitlabToken'
+Use /nc secret_create to create keys
+Visit https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html to learn how to setup a Personal Access Token on GitLab
+```
+
+## Usage
+Viewing community contributions for a GitLab project.
+```
+/dapp gitlab_stats <repo>
+```
+![](https://raw.githubusercontent.com/Maljean/command-sets/master/gitlab/screenshots/statsCommandScreenshot.PNG)
```

Comment:
Should the github url be updated to point to `command-set` repo instead?

Reply:
It should! Last week I was very hesitant to guess it. Now that I know this will be on the master branch I can update it.